### PR TITLE
EXLM-1579 Awards Block UE & CSS Changes

### PR DESCRIPTION
**Pull request by mayukhs2904**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1579

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page
- After: https://feat-exlm-1579--exlm--adobe-experience-league.hlx.page

### CODE REVIEW

File: component-definition.json [hunk 1]
```diff
@@ -1015,6 +1015,20 @@
               }
             }
           }
+        },
+        {
+          "title": "Awards Block",
```
Comment (ruhisingh1) [inline component-definition.json:1020]: Rename it to just "Awards"

**Review by ruhisingh1 — DISMISSED** [review]

File: blocks/awards/awards.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    margin-bottom: 8px;
+    border-bottom: 2px var(--color-white) solid;
+    padding: 0 0 8px 0;
+}
+
+.awards-description {
+    font-size: var(--spectrum-font-size-200);
+}
+
+.awards-noAwardsYet {
```
Comment (noormd08) [inline blocks/awards/awards.css:35]: Don’t use camelcase

File: blocks/awards/awards.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    .awards-holder {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+@media(min-width: 900px) {
+    .awards-holder {
+        grid-template-columns: repeat(3, 1fr);
+    }
+}
```
Comment (noormd08) [inline blocks/awards/awards.css:50]: Keep one parent reference in the css. So that there won’t be any conflicts in future.

**Review by ruhisingh1 — APPROVED** [review]

File: blocks/awards/awards.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    margin-bottom: 8px;
+    border-bottom: 2px var(--color-white) solid;
+    padding: 0 0 8px 0;
+}
+
+.awards .awards-description {
+    font-size: var(--spectrum-font-size-200);
+}
+
+.awards .awards-no-award-yet {
```
Comment (noormd08) [inline blocks/awards/awards.css:35]: keep the class names as nil-awards

File: blocks/awards/awards.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+try {
+  placeholders = await fetchLanguagePlaceholders();
+} catch (err) {
+  // eslint-disable-next-line no-console
+  console.error('Error fetching placeholders:', err);
+}
+
+const COMPLETED = placeholders?.completed || 'COMPLETED';
+const NO_AWARDS_YET =
+  placeholders?.noAwardsYet || 'No awards yet! Start exploring Experience League to discover what you can earn.';
```
Comment (noormd08) [inline blocks/awards/awards.js:15]: Please remove the declaration from the top and use it directly, as we are not reusing this variable in multiple places

**Review by noormd08 — DISMISSED** [review]

**Review by noormd08 — APPROVED** [review]

**Pull request merged**